### PR TITLE
recommendate vscode extension

### PR DESCRIPTION
**Pull request by Shinigami92**
- [x] I have read the [Contributor Guide](https://github.com/nteract/nteract/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "eamodio.gitlens",
+    "esbenp.prettier-vscode",
```
Comment (rgbkrk) [inline .vscode/extensions.json:9]: 👍

File: .vscode/extensions.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  // See http://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "editorconfig.editorconfig",
```
Comment (rgbkrk) [inline .vscode/extensions.json:7]: 👍

File: .vscode/extensions.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    "eamodio.gitlens",
```
Comment (rgbkrk) [inline .vscode/extensions.json:8]: I'm not sure if we should force the gitlens one on folks.

Comment (Shinigami92) [reply .vscode/extensions.json:8]: This are recomments not "u must install this or close the project" ^^
But if you want I can remove this line when I'm back from work

**Comment by rgbkrk** [comment]
Amusingly I already installed the extensions.

**Pull request merged**